src/test/java/com/exasol/adapter/dialects/athena/AthenaIdentifierTest.java

```diff
@@ -0,0 +1,29 @@
+package com.exasol.adapter.dialects.athena;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import nl.jqno.equalsverifier.EqualsVerifier;
+
+class AthenaIdentifierTest {
+    @ParameterizedTest
+    @ValueSource(strings = { "_myunderscoretable", "123columnone", "テスト", "таблица" })
+    void testCreateValidIdentifier(final String identifier) {
+        assertDoesNotThrow(() -> AthenaIdentifier.of(identifier));
+    }
```

Comment:
Is Unicode really supported? Documentation doesn't say anything about it, I think.

Reply:
I've googled some Japanese articles where people said that it's possible to use Japanese symbols in the Athena names, but some errors can occur. So I decided to let them be